dls-controls/malcolmjs
# Feature/table widget #156

## Description

Added initial structure for table widget and refactored attributeSelector to have a separate function for selection of widget given an input tag (to be used by the table). Also includes bugfix for popOut window.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm run storybook
- view table widget examples in storybook

## Agile board tracking

connect to #156 
connect to #157 
connect to #158 
connect to #159 
connect to #160
